src/main/java/com/easypost/model/Shipment.java

```diff
@@ -789,32 +789,32 @@ public Shipment newRates(final Map<String, Object> params, final String apiKey)
     /**
      * Get smart rates for this Shipment.
```

Comment:
These should all be one word please

```suggestion
     * Get smartrates for this Shipment.
```